libobs/obs-source.h

```diff
@@ -452,6 +462,16 @@ struct obs_source_info {
 	 * @return          The properties data
 	 */
 	obs_properties_t *(*get_properties2)(void *data, void *type_data);
+
+	void (*media_play_pause)(void *data);
```

Comment:
Instead of a `play_pause` function that presumably toggles the play/pause state, have it set whether you want to pause or unpause.  For example:

```C
	void (*media_set_paused)(void *data, bool pause);
```

And then do the same for the actual source function as well if possible.  Not sure if it's worth adding a function to get the current pause state considering you already have a function to get the state, `media_get_state`.

Reply:
The VLC function to play and pause is a toggle.